--- a/PKGBUILD
+++ b/PKGBUILD
@@ -10,7 +10,7 @@
 arch=('any')
 url='https://jenkins.io'
 license=('MIT')
-depends=('java-runtime=8' 'ttf-dejavu')
+depends=('fontconfig' 'java-runtime=11' 'ttf-dejavu')
 provides=('jenkins-ci')
 conflicts=('jenkins-ci')
 replaces=('jenkins-ci')
@@ -31,6 +31,10 @@
             '896cbc64eadb2f3bb9fa7fc26dbfc527b1823d8bff3158be7f1936cc250f55eeafe89079e1ceb8f3d269041662573ccbbb36248140b4ba24144a564366f8847a'
             '0df6479a600db395c12ad29ebda3120d00460338b9a6b15913e2245aa29212a3798c32e664ca70c8034146e2e0b573066deec9e3210198202f753994e1b83efa')

+prepare() {
+  patch --forward --strip=1 --input="${srcdir}/../../jenkins.conf.patch"
+}
+
 package() {
   install -Dm444 LICENSE "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
   install -Dm444 "jenkins-${pkgver}.war" "$pkgdir/usr/share/java/jenkins/jenkins.war"
